opencti-worker/src/push_handler.py

```diff
@@ -73,43 +104,73 @@ def handle_message(
+                            self.pika_parameters
+                        )
+                        push_channel = push_pika_connection.channel()
+                        try:
+                            push_channel.confirm_delivery()
+                        except Exception as err:  # pylint: disable=broad-except
+                            self.logger.warning(str(err))
+                        for too_large_item_bundle in too_large_items_bundles:
+                            self.logger.warning(
+                                "Detected a bundle too large, sending it to dead letter queue...",
+                            )
+                            self.send_bundle_to_specific_queue(
+                                push_channel,
+                                self.listen_exchange,
+                                self.listen_too_large_routing,
+                                data,
+                                too_large_item_bundle,
+                            )
                 else:
                     # As bundle is received as complete, split and requeue
                     # Create a specific channel to push the split bundles
-                    with pika.BlockingConnection(self.pika_parameters) as push_pika_connection:
+                    with pika.BlockingConnection(
+                        self.pika_parameters
+                    ) as push_pika_connection:
                         with push_pika_connection.channel() as push_channel:
                             try:
                                 push_channel.confirm_delivery()
                             except Exception as err:  # pylint: disable=broad-except
                                 self.logger.warning(str(err))
                             # Instance spliter and split the big bundle
                             event_version = content.get("x_opencti_event_version")
-                            stix2_splitter = OpenCTIStix2Splitter()
-                            expectations, _, bundles = (
+                            stix2_splitter = OpenCTIStix2Splitter(self.objects_max_refs)
+                            expectations, _, bundles, too_large_elements_bundles = (
                                 stix2_splitter.split_bundle_with_expectations(
                                     content, False, event_version
                                 )
                             )
                             # Add expectations to the work
                             if work_id is not None:
                                 self.api.work.add_expectations(work_id, expectations)
+                                # For each bundle too large, send it to the too large queue
+                            for too_large_elements_bundle in too_large_elements_bundles:
+                                self.logger.warning(
+                                    "Detected a bundle too large, sending it to dead letter queue...",
+                                )
```

Comment:
could we log more information about the rejected bundle ?
or maybe it's already covered by python client lots ?